resources/webdid/test-dataspace-authority/did.json

```diff
@@ -0,0 +1,33 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    {
+      "@base": "did:web:mvd215dataspacedid.z16.web.core.windows.net"
+    }
+  ],
+  "authentication": [
+    "#identity-key-authority"
+  ],
+  "id": "did:web:mvd215dataspacedid.z16.web.core.windows.net",
+  "service": [
+    {
+      "id": "#enrollment-url",
```

Comment:
nit: needs to be updated as we changed it in MVD PR https://github.com/agera-edc/MinimumViableDataspace/pull/203/files